docs/pt_BR/README.md

```diff
@@ -0,0 +1,82 @@
+![](../images/logo.png)
+
+> Um template para criar aplicações electron usando vue \(como o nome diz\).
+
+[![Build Status](https://semaphoreci.com/api/v1/simulatedgreg/electron-vue/branches/master/badge.svg)](https://semaphoreci.com/simulatedgreg/electron-vue)
+
+[![js-standard-style](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://github.com/feross/standard)
+
+[![forthebadge](http://forthebadge.com/images/badges/built-with-love.svg)](http://forthebadge.com) [![forthebadge](http://forthebadge.com/images/badges/uses-js.svg)](http://forthebadge.com) [![forthebadge](http://forthebadge.com/images/badges/makes-people-smile.svg)](http://forthebadge.com)
+
+## Resumo
+
+O foco do template é remover a nescessidade de criar manualmente todo a organização do projeto usando electron e o vue. electron-vue tira vantagem da `vue-cli` para receber arquivos, `webpack` com `vue-loader`, `electron-packager` ou `electron-builder`, e também os plugins mais utilizados como `vue-router`, `vuex` e muito mais.
```

Comment:
- Change `nescessidade` to `necessidade`
- Change `manualmente todo a` to `manualmente toda a`